### ISSUE TITLE
Accessible authentication - consider other phrase than one time password

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3180]
In discussion around approving a PR to address https://github.com/w3c/wcag/issues/3095
there was considerable discussion around passcode and password, and then someone suggested verification code

I dug in to usage trends and the latter looks more solid. I will continue investigation and create a small PR
https://trends.google.com/trends/explore?date=all&q=one%20time%20passcode,one%20time%20password,verification%20code

**Comment by johnsherrod** [comment]
Common observed phrases for 2FA are "temporary code", "verification code", "security code", "advanced access code" but I rarely see "password" or "passcode" as it applies to authentication.  "verification code" seems very aligned with intent as a more clear and understandable option.

**Comment by patrickhlauke** [comment]
related: this has now been merged https://github.com/w3c/wcag/pull/3150 so there is now a "section specifically addressing the use of verification codes" https://w3c.github.io/wcag/understanding/accessible-authentication-minimum.html#two-factor-authentication as a starting point

**Comment by mbgower** [comment]
I have made modifications to the newly updated Understanding document, as well as the recent Failure technique.

Please review the PR and feel free to comment here or there for suggested changes. https://github.com/w3c/wcag/pull/3185
